### PR TITLE
Adapt csv and remove scorecard section

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

File: config/olm/kubernetes/bundle-0.2.1.Dockerfile [hunk 1]
```diff
@@ -10,10 +10,6 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.5.0
 LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 
-# Labels for testing.
-LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
```
Comment (chrismuellner) [inline config/olm/kubernetes/bundle-0.2.1.Dockerfile:14]: Should this also be removed for `0.2.2`?

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: config/olm/kubernetes/0.1.0/bundle.Dockerfile [hunk 2]
```diff
@@ -9,7 +9,5 @@ LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.3.0
```
Comment (meik99) [inline config/olm/kubernetes/0.1.0/bundle.Dockerfile:9]: the latest operator sdk is 1.13.0, please double check that this value is correct

File: config/olm/kubernetes/0.1.0/bundle/metadata/annotations.yaml [hunk 3]
```diff
@@ -8,5 +8,3 @@ annotations:
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.3.0
```
Comment (meik99) [inline config/olm/kubernetes/0.1.0/bundle/metadata/annotations.yaml:8]: the latest operator sdk is 1.13.0, please double check that this value is correct

File: config/olm/kubernetes/0.2.1/manifests/dynatrace-operator.v0.2.1.clusterserviceversion.yaml [hunk 4]
```diff
@@ -40,7 +40,7 @@ metadata:
       ]
     capabilities: Deep Insights
     categories: Monitoring,Logging & Tracing
-    containerImage: docker.io/dynatrace/dynatrace-operator
+    containerImage: docker.io/dynatrace/dynatrace-operator:v0.2.1
     operators.operatorframework.io/builder: operator-sdk-v1.5.0
```
Comment (meik99) [inline config/olm/kubernetes/0.2.1/manifests/dynatrace-operator.v0.2.1.clusterserviceversion.yaml:44]: the latest operator sdk is 1.13.0, please double check that this value is correct

File: config/olm/kubernetes/0.2.1/metadata/annotations.yaml [hunk 5]
```diff
@@ -9,7 +9,3 @@ annotations:
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.5.0
```
Comment (meik99) [inline config/olm/kubernetes/0.2.1/metadata/annotations.yaml:10]: the latest operator sdk is 1.13.0, please double check that this value is correct

File: config/olm/kubernetes/bundle-0.2.1.Dockerfile [hunk 6]
```diff
@@ -10,10 +10,6 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.5.0
```
Comment (meik99) [inline config/olm/kubernetes/bundle-0.2.1.Dockerfile:10]: the latest operator sdk is 1.13.0, please double check that this value is correct

File: config/olm/openshift/0.2.1/metadata/annotations.yaml [hunk 7]
```diff
@@ -10,6 +10,3 @@ annotations:
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.5.0
```
Comment (meik99) [inline config/olm/openshift/0.2.1/metadata/annotations.yaml:10]: the latest operator sdk is 1.13.0, please double check that this value is correct

**Review by meik99 — CHANGES_REQUESTED** [review]

**Comment by chrismuellner** [comment]
Scorecards will be removed automatically for future releases

**Pull request closed without merging**